bookwyrm/models/user.py

```diff
@@ -128,6 +128,13 @@ class User(OrderedCollectionPageMixin, AbstractUser):
     show_suggested_users = models.BooleanField(default=True)
     discoverable = fields.BooleanField(default=False)
 
+    # feed options
+    feed_status_types = ArrayField(
+        models.CharField(max_length=10, blank=False),
+        size=8,
+        default=list(["review", "comment", "quotation", "everything"]),
```

Comment:
On app start I get the following warning, concerning this list:

```
bookwyrm-web-1             | WARNINGS:
bookwyrm-web-1             | bookwyrm.User.feed_status_types: (fields.E010) ArrayField default should be a callable instead of an instance so that it's not shared between all field instances.
bookwyrm-web-1             | 	HINT: Use a callable instead, e.g., use `list` instead of `[]`.
```

Reply:
You can use choices to simply a few things here. First, you can define a choice block that has the translations in it like you have in the multi select choices:
``` python3
FeedFilterChoices = [
    "review", _("Reviews"),
    ...
]
```

And then you can define `choices` in the `ArrayField`, which will enforce the expected values:

```python3
ArrayField(
    models.CharField(max_length=10, blank=False),
    choices=FeedFilterChoices,
    ...
)
```

And finally, define a function that returns the default list so that it isn't grumpy about that:
```python3
def get_feed_filter_choices():
    return [f[0] for f in FeedFilterChoices]

...

ArrayField(
    models.CharField(max_length=10, blank=False),
    choices=FeedFilterChoices,
    default=get_feed_filter_choices,
    ...
)
```

I think if the model field is declared this way, you won't have to pass `choices` to `widgets.CheckboxSelectMultiple`.